### PR TITLE
fix: Removed npm usage

### PR DESCRIPTION
**Pull request by Laegel**

### CODE REVIEW

**Comment by lucasfernog** [comment]
I guess we can't just force everyone to use yarn, we should teach them to add a `tauri` script on package.json and `npm run` it instead of using npx.

**Comment by nothingismagick** [comment]
No, not npm, npx

File: docs/usage/development/integration.md [hunk 1]
```diff
@@ -30,6 +30,20 @@ npm install tauri --save
   You can install Tauri as both a local and a global dependency, but we recommend installing it locally.
 </Alert>
 
+If you decide to use Tauri as a local package with npm, you will _have to_ define custom scripts to your package.json:
+
+```js title=package.json
+{
+  // This content is just a sample
+  "scripts": {
+    "tauri:init": "tauri init",
```
Comment (lucasfernog) [inline docs/usage/development/integration.md]: I think we just leave a `"tauri": "tauri"` script then the user can run `npm run tauri init` `npm run tauri build`.

**Review by lucasfernog — APPROVED** [review]

**Comment by CreativeCactus** [comment]
Is there any reason not to use `npx tauri`? Getting started commands all work fine for me as `npx tauri ...`.

**Comment by nothingismagick** [comment]
We've had very serious problems with npx recently, including it actually using an outdated version. If it works for you, that's great, however we won't promote it again until all security issues are resolved. (also it's slow).

**Pull request merged**